app/helpers/audits_helper.rb

```diff
@@ -234,6 +234,13 @@ def construct_additional_info(audits)
     end
   end
 
+  def audits_metadata(audits)
+    {
+      :audits => construct_additional_info(audits),
+      :itemCount => audits.count,
+    }
+  end
```

Comment:
in plugins this could be override, e.g:
```rb
module PluginAuditsHelper
  def audits_metadata(audits)
    super.merge(
      documentationUrl: '/some_other_url'
    )
  end
end
```